channels/rdpecam/client/encoding.c

```diff
@@ -24,12 +24,139 @@
 
 #define TAG CHANNELS_TAG("rdpecam-video.client")
 
+#if defined(WITH_INPUT_FORMAT_H264)
+/*
+ * demux a H264 frame from a MJPG container
+ * args:
+ *    srcData - pointer to buffer with h264 muxed in MJPG container
+ *    srcSize - buff size
+ *    h264_data - pointer to h264 data
+ *    h264_max_size - maximum size allowed by h264_data buffer
+ *
+ * Credits:
+ *    guvcview    http://guvcview.sourceforge.net
+ *    Paulo Assis <pj.assis@gmail.com>
+ *
+ * see Figure 5 Payload Size in USB_Video_Payload_H 264_1 0.pdf
+ * for format details
+ *
+ * @return: data size and copies demuxed data to h264 buffer
+ */
+static size_t demux_uvcH264(const BYTE* srcData, size_t srcSize, BYTE* h264_data,
+                            size_t h264_max_size)
+{
+	WINPR_ASSERT(h264_data);
+	WINPR_ASSERT(srcData);
+
+	const uint8_t* sp = NULL;
+	const uint8_t* spl = NULL;
+	const uint8_t* epl = NULL;
+	const uint8_t* header = NULL;
```

Comment:
if possible declare variables at first use, only if you need it initialized before (`goto` et al) at beginning